src/Entity/Transaction.php

```diff
@@ -7,7 +7,7 @@
 
 /**
  * @ORM\Entity(repositoryClass="AppBundle\Repository\TransactionRepository")
- * @ORM\Table(indexes={
+ * @ORM\Table(name="donation_transactions", indexes={
  *     @ORM\Index(name="transaction_result_idx", columns={"paybox_result_code"})
```

Comment:
To be consistent with the renaming: `transaction_result_idx` => `donation_transaction_result_idx`